core/primer.md

```diff
@@ -2,6 +2,185 @@
 
 <!-- no verify-specs -->
 
+## Abstract
+
+This non-normative document provides an overview of the xRegistry specification.
```

Comment:
It might be nice to turn one of the "xRegistry" references in here (perhaps the first one) into a link to the spec itself.